src/github.com/getlantern/flashlight/statreporter/statreporter.go

```diff
@@ -196,8 +208,22 @@ func (dgAccum *dimGroupAccumulator) makeReport() report {
 		"dims": dgAccum.dg.dims,
 	}
 
-	for category, accum := range dgAccum.categories {
-		report[category] = accum
+	for category, s := range dgAccum.categories {
+		if category == members {
+			// Transform maps into arrays
+			s2 := make(stats)
+
+			for k, v := range s {
+				m := v.(map[string]bool)
+				a := make([]string, 0, len(m))
+				for member, _ := range m {
```

Comment:
You could actually drop the blank identifier here since the value doesn't have to be accounted for.. 
for member := range m { ... }